locales/nb_NO/pcsx2_Main.po

```diff
@@ -1198,33 +1134,31 @@ msgstr "Velg en mappe for %sinnstillinger"
 msgid "Settings"
 msgstr "Innstillinger"
 
-#: pcsx2/gui/Dialogs/FirstTimeWizard.cpp:86
+#: pcsx2/gui/Dialogs/FirstTimeWizard.cpp:84
 msgid "Language selector"
 msgstr "Språkvelger"
 
-#: pcsx2/gui/Dialogs/FirstTimeWizard.cpp:89
+#: pcsx2/gui/Dialogs/FirstTimeWizard.cpp:87
 msgid ""
 "Change the language only if you need to.\n"
 "The system default should be fine for most operating systems."
 msgstr ""
 "Skift språket bare hvis du trenger det.\n"
-"Systemet sin standardinnstilling bør gå greit for de fleste operativsystemer."
+"Systemet sin standardinnstilling burde være greit for de fleste operativsystemer."
 
-#: pcsx2/gui/Dialogs/FirstTimeWizard.cpp:95
+#: pcsx2/gui/Dialogs/FirstTimeWizard.cpp:93
 msgid "Welcome to PCSX2!"
 msgstr "Velkommen til PCSX2!"
 
-#: pcsx2/gui/Dialogs/FirstTimeWizard.cpp:106
-#, fuzzy
-msgid "Configuration Guide"
-msgstr "Oppsettsanvisninger (Online)"
+#: pcsx2/gui/Dialogs/FirstTimeWizard.cpp:104
+msgid "Configuration Guides (online)"
+msgstr "Oppsettsanvisninger (online)"
 
-#: pcsx2/gui/Dialogs/FirstTimeWizard.cpp:110
-#, fuzzy
-msgid "Readme / FAQ"
+#: pcsx2/gui/Dialogs/FirstTimeWizard.cpp:108
+msgid "Readme / FAQ (Offline/PDF)"
```

Comment:
Ditto, but `Readme / FAQ`.

Reply:
Wait. I did translate those. They still are translated in main.po. 
Just checked, the translation doesn't show in PCSX2 on the latest(ish) development build.

Reply:
I think you translated the old version of the string because you were working from an older git revision. That's why the translations aren't appearing for these two strings.

Reply:
crap...
Well, going to do that tomorrow then.